packages/lit-virtualizer/README.md

```diff
@@ -121,31 +121,53 @@ render(templateResult, document.querySelector("#post-list"));
 
 In this example, just `items` and `template` were configured. You can also specify the scroll target and whether or not to use shadow DOM.
 
+
+
 ### `scroll` config options
 
-`items: Array<T>`
+#### `items`
+
+Type: `Array<T>`
+
+A list of items to display via the template function. The type of the items should match the first argument of the template function.
+
+#### `template`
+
+Type: `(item: T, index?: number) => lit-html.TemplateResult`
+
+A function that returns a lit-html TemplateResult. It will be used to generate the DOM for each item in the virtual list.
+
+#### `scrollTarget`
+
+Type: `Element | Window`
+
+Optional. An element that receives scroll events for the virtual scroller. If not specified, the directive's parent element will be the target.
+
+#### `useShadowDOM`
 
-> A list of items to display via the template function. The type of the items should match the first argument of the template function.
+Type: `boolean`
 
-`template: (item: T, index?: number) => lit-html.TemplateResult`
+Optional. Whether to build the virtual scroller within a shadow DOM.
 
-> A function that returns a lit-html TemplateResult. It will be used to generate the DOM for each item in the virtual list.
+#### `totalItems`
 
-`scrollTarget: Element | Window`
+Type: `number`
 
-> Optional. An element that receives scroll events for the virtual scroller. If not specified, the directive's parent element will be the target.
+Optional. Limit for the number of items to display. Defaults to the length of the items array.
```

Comment:
Suggest:

Optional. An integer representing the actual or estimated number of items in the entire data set.

If all of the data is present in the `items` array, then you can omit this property and the array's length will be used.

If you initially load only part of the data into the `items` array, then you can reduce "jumpiness" in the size and position of the scroll thumb by setting this property to the actual number of items in the data set (including items not yet loaded).

Reply:
Ahh, I see! Will add.

Reply:
I'll rename this in a separate PR/commit to keep this one consistent.